src/seedu/addressbook/Main.java

```diff
@@ -109,10 +116,14 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (FileNotFoundException fnfe) {
+        	ui.showToUser(fnfe.getMessage());
```

Comment:
Also consider creating the file and show the problem to the user, instead of closing the program.